src/main/drivers/bus_spi.c

```diff
@@ -278,7 +286,7 @@ bool spiWriteRegister(const busDevice_t *bus, uint8_t reg, uint8_t data)
     return true;
 }
 
-bool spiReadRegisterBuffer(const busDevice_t *bus, uint8_t reg, uint8_t length, uint8_t *data)
+bool spiBusReadRegisterBuffer(const busDevice_t *bus, uint8_t reg, uint8_t *data, uint8_t length)
```

Comment:
spiBusReadRegisters will work here too.. It may be easier to understand and function signature should catch typos..